From 335fc37f672d123453b0481c6316492ab2a7d18a Mon Sep 17 00:00:00 2001
From: Rob Rodriguez <rrodrigu@akamai.com>
Date: Tue, 18 Aug 2020 07:29:36 -0700
Subject: [PATCH] adding check for at least one provider state set

---
 pkg/akamai/provider.go | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pkg/akamai/provider.go b/pkg/akamai/provider.go
index 1262c04..318d224 100644
--- a/pkg/akamai/provider.go
+++ b/pkg/akamai/provider.go
@@ -130,9 +130,8 @@ func Provider(log hclog.Logger, provs ...Subprovider) plugin.ProviderFunc {
 		}
 
 		instance.ConfigureContextFunc = func(ctx context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
-			log.Debug("START providerConfigure  %s", instance.TerraformVersion)
+			var stateSet bool
 
-			stateSet := false
 			for _, p := range instance.subs {
 				state, err := p.Configure(ctx, log, d)
 				if err != nil {
-- 
2.25.1

